styles.css

```diff
@@ -1288,6 +1292,8 @@ ul {
 .btn:active {
```

Comment:
Kindly follow figma design colors and effects as requested by the previous reviewer as your button doesn't change color when we hover  over it also when pressed the color is not as it is in figma, you have done a great work in this project and you need just to stick to your template design as the goal in this activity is to replicate the given design 😊

![temp1 button effects](https://user-images.githubusercontent.com/59707859/220620131-86e2b3f4-581f-418e-b65b-c01b15ee4d3d.PNG)